### ISSUE TITLE
Error on empty translation in fallbackLanguage

### ISSUE THREAD

**Issue by vitexikora** [lingui/js-lingui#51]
Empty translation (`"translation": "",`) in language set as fallbackLanguage throws
```js
if (input.charCodeAt(peg$currPos) === 123) {
             ^

TypeError: Cannot read property 'charCodeAt' of undefined
```

**Comment by tricoder42** [comment]
Is it during compile or extract?

**Comment by vitexikora** [comment]
during compile

**Comment by benbender** [comment]
I can confirm the bug. 

`/Users/bb/Development/test-setup/node_modules/messageformat-parser/parser.js:459
    if (input.charCodeAt(peg$currPos) === 123) {
             ^

TypeError: Cannot read property 'charCodeAt' of undefined
    at peg$parseargument (/Users/bb/Development/test-setup/node_modules/messageformat-parser/parser.js:459:14)
    at peg$parsetoken (/Users/bb/Development/test-setup/node_modules/messageformat-parser/parser.js:408:10)
    at peg$parsestart (/Users/bb/Development/test-setup/node_modules/messageformat-parser/parser.js:396:10)
    at peg$parse (/Users/bb/Development/test-setup/node_modules/messageformat-parser/parser.js:1815:16)
    at compile (/Users/bb/Development/test-setup/node_modules/lingui-cli/dist/api/compile.js:60:16)`

The error disappears if I set the fallbackLanguage to an empty string.

**Comment by tricoder42** [comment]
Fixed in lingui-cli@**1.0.5**
